swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/AbstractResourceTypeHandler.kt

```diff
@@ -437,7 +437,7 @@ abstract class AbstractResourceTypeHandler<T : Resource>(
   ) {
     if (markedResource != null && !workConfiguration.dryRun) {
       try {
-        log.info("{} is no longer a candidate because: $reason.", markedResource.uniqueId())
+        log.debug("{} is no longer a candidate because: $reason.", markedResource.uniqueId())
```

Comment:
This would be useful , right ?

Reply:
this message hasn't been actually that useful in practice, I think.